From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Piotr Szczepanik <piter75@gmail.com>
Date: Mon, 4 Jan 2021 01:02:20 +0100
Subject: [ARCHEOLOGY] Switched rockchip64/rk3399 to u-boot v2020.10 (#2512)

> X-Git-Archeology: > recovered message: > * Switched rockchip64/rk3399 to u-boot v2020.10
> X-Git-Archeology: > recovered message: > * Some tweaks for SPI booting on ROCK Pi 4 in u-boot v2020.10
> X-Git-Archeology: > recovered message: > * Adjusted NanoPi M4V2 config for u-boot v2020.10
> X-Git-Archeology: > recovered message: > * Disabled PREBOOT for RockPro64 in u-boot v2020.10
> X-Git-Archeology: > recovered message: > It breaks boot during fdt loading
> X-Git-Archeology: > recovered message: > * Fix issues with booting from SPI flash for RockPro64
> X-Git-Archeology: > recovered message: > * Moved NanoPi R4S patches to board specific folder
> X-Git-Archeology: - Revision 2a3c216d4107ddf30abadc6ba3c5707c1fa59ff2: https://github.com/armbian/build/commit/2a3c216d4107ddf30abadc6ba3c5707c1fa59ff2
> X-Git-Archeology:   Date: Mon, 04 Jan 2021 01:02:20 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched rockchip64/rk3399 to u-boot v2020.10 (#2512)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 86abac1fd4e169712a44d245cf7adef4ee17c420: https://github.com/armbian/build/commit/86abac1fd4e169712a44d245cf7adef4ee17c420
> X-Git-Archeology:   Date: Tue, 26 Oct 2021 22:14:41 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Cleanup u-boot configurations for rockchip64 derivatives (#3150)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 26437e36c18bb09484f4150e396a1784cc6471b7: https://github.com/armbian/build/commit/26437e36c18bb09484f4150e396a1784cc6471b7
> X-Git-Archeology:   Date: Thu, 16 Jun 2022 12:27:05 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched rockchip64 u-boot to v2022.04 (#3871)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f99b356ff47757f59bee370a6c2b0329b0a028e6: https://github.com/armbian/build/commit/f99b356ff47757f59bee370a6c2b0329b0a028e6
> X-Git-Archeology:   Date: Fri, 14 Oct 2022 09:19:50 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Bump Rockchip64 u-boot to 2022.07 (#4246)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 configs/rockpro64-rk3399_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/rockpro64-rk3399_defconfig b/configs/rockpro64-rk3399_defconfig
index 3f24728f20..9d6d60c669 100644
--- a/configs/rockpro64-rk3399_defconfig
+++ b/configs/rockpro64-rk3399_defconfig
@@ -14,7 +14,6 @@ CONFIG_SPL_SPI_FLASH_SUPPORT=y
 CONFIG_SPL_SPI=y
 CONFIG_DEBUG_UART=y
 CONFIG_SYS_LOAD_ADDR=0x800800
-CONFIG_USE_PREBOOT=y
 CONFIG_DEFAULT_FDT_FILE="rockchip/rk3399-rockpro64.dtb"
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_MISC_INIT_R=y
-- 
Armbian

